docs/ml-datasource.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: global
+title: Data sources
+displayTitle: Data sources
+---
+
+In this section, we introduce how to use data source in ML to load data.
+Beside some general data sources such as Parquet, CSV, JSON and JDBC, we also provide some specific data source for ML.
+
+**Table of Contents**
+
+* This will become a table of contents (this text will be scraped).
```

Comment:
is this convention, to have this text here in the table of contents? "* This will become a table of contents (this text will be scraped)."

Reply:
Yes. This keep the same with other ML algo page.

Reply:
ah, ok, great